requirements.txt

```diff
@@ -1,5 +1,5 @@
 aiohttp
 ujson
 aiogram
-pytube3
+git+https://github.com/nficano/pytube
```

Comment:
if you just write the dependency as "pytube" then it won't work? Is the full URL necessary here?

Reply:
from the pytube [README](https://github.com/nficano/pytube)

> To install from pypi with pip:
> `$ python -m pip install pytube`
> Sometime, the pypi release becomes slightly outdated. To install from the source with pip:
> `$ python -m pip install git+https://github.com/nficano/pytube`

seems pypi fetch tags to build it
https://pypi.org/project/pytube/#history

since pytube is usually broken because of youtube changes, I think it's better to use the git version.
fwiw, you may need to `pip uninstall pytube` and then `pip install git+https://github.com/nficano/pytube` to reinstall when you need to update it.

seems they tag quite often, maybe it's better to switch to pypi version. I'll do it.